app/Http/Controllers/UserController.php

```diff
@@ -112,10 +123,9 @@ public function edit($id)
     {
         /** @var User $user */
         $user = $this->userRepository->findOrFail($id);
-
         $userObj = $user->toArray();
         $userObj['project_ids'] = $user->projects()->pluck('project_id')->toArray();
-
+        $userObj['role_id']=$user->roles()->pluck('role_id')->toArray();
```

Comment:
the code should be formatted.
```suggestion
        $userObj['role_id'] = $user->roles()->pluck('role_id')->toArray();
```